utils/check_dummies.py

```diff
@@ -165,7 +165,7 @@ def {0}(*args, **kwargs):
 
 
 def read_init():
-    """ Read the init and exctracts PyTorch, TensorFlow, SentencePiece and Tokenizers objects. """
+    """ Read the init and extracts PyTorch, TensorFlow, SentencePiece and Tokenizers objects. """
```

Comment:
either "extract"  (or "reads and extracts") -  a tiny correction of the correction